docs/decommission.rst

```diff
@@ -0,0 +1,80 @@
+Decommission SecureDrop
+=======================
+
+The following steps will guide you through the decommissioning of your
+SecureDrop instance.
+
+#. **Put a notice in advance on your landing page to inform sources that your
+   instance will soon be retiring.**
+   You may want to direct them to other secure methods of contacting you.
+#. **Locate and create an inventory of all your hardware.**
+   - *Journalist Workstation* USBs
+   - *Admin Workstation* USB(s)
+   - *Secure Viewing Station* USB
+   - *Secure Viewing Station* computer
+   - Transfer and Export devices (USBs, optical drives, or external drives)
+   - Backup USBs/other storage media
+   - Servers
+   - Firewall
+
+   .. note:: The recommended SecureDrop setup includes only one *Secure Viewing
+      Station* USB. However, if you have been working remotely or
+      have a non-standard setup, you may have more than one *SVS USB*. It is
+      important that you locate all of these USBs, since they hold the most
+      sensitive data.
+#. **Decide whether to save a backup.**
```

Comment:
s/Decide whether to/ Optional: /

Reply:
fixed in f38127 and all optional steps formatted this way for consistency